### ISSUE TITLE
Edit option appers only the second time I visit the issue.

### ISSUE THREAD

**Issue by KOLANICH** [jonan/ForkHub#328]
To make the ```edit``` option appear I have to tap back and tap that issue again.

**Comment by KOLANICH** [comment]
1 I create an issue.
2 some time passed. I guess that if I just restart (manually unloading) the app, the effect should be the same.
3 the app is started, I navigate to that issue, for example use search in the repo or notifications
4 In the issue I don't see ```edit``` option in the menu.
5 I tap back button
6 I tap on the issue list item again. The issue is reopened, ```edit``` menu item is present.

**Comment by licaon-kter** [comment]
Yep, just repro it, when following the issue number in #327 to #326.
